CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ numbering uses [semantic versioning](http://semver.org).
 - Add option to install Service Preview components (traffic-manager, traffic-agent).
 - Add option to install ambassador-injector, alongside Service Preview.
 
+## v6.4.1
```

Comment:
A few lines up, there are changelog items for "Next Release". These should be moved under v6.4.1.

Reply:
*sigh* this should actually be under 6.4.0.

That release was never actually pushed and I updated the changelog incorrectly.

Reply:
Thanks!